cpu/nrf51822/radio/nrfmin/nrfmin.c

```diff
@@ -0,0 +1,730 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_nrf51822_nrfmin
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the nrfmin NRF51822 minimal radio driver
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "mutex.h"
+#include "thread.h"
+#include "sched.h"
+#include "kernel.h"
+#include "periph_conf.h"
+#include "periph/cpuid.h"
+#include "nrfmin.h"
+#include "net/ng_netbase.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/**
+ * @brief   Driver specific device configuration
+ * @{
+ */
+#define CONF_MODE               RADIO_MODE_MODE_Nrf_2Mbit
+#define CONF_PAYLOAD_LEN        (250U)
+#define CONF_LEN                (8U)
+#define CONF_S0                 (0U)
+#define CONF_S1                 (0U)
+#define CONF_STATLEN            (0U)
+#define CONF_BASE_ADDR_LEN      (4U)
+#define CONF_ENDIAN             RADIO_PCNF1_ENDIAN_Big
```

Comment:
In "Reference manual v3.0 17.1.13" is documented using big endian disables "Device Address Match" feature. This feature looks useful for addressing.